.golangci.yml

```diff
@@ -64,7 +64,10 @@ linters-settings:
     go: "1.18"
   stylecheck:
     go: "1.18"
+  nolintlint:
+    allow-unused: true # required since go 1.18
   depguard:
+    list-type: denylist # required since go 1.18
```

Comment:
These are required since updating the `go.mod` to 1.18 defaults them to a different value. I didn't spend time updating the golanci-lint configuration given that we can pull in upstream changes later.